techniques/fileDistribution/fileTemplate/1.0/metadata.xml

```diff
@@ -19,7 +19,7 @@
 
```

Comment:
We also need to specify that the technique support the generation of multiple policy without merge. 
For now, the xml element for that is: 
  <POLICYGENERATION>multi</POLICYGENERATION>

(and a default value of "merge" in place of multi, if the tag is missing, for compatibility)

Reply:
I'd replace "multi" with "separate"